### PR TITLE
adding mongodb as destination ( data integration )

### PR DESCRIPTION
**Pull request by Luishfs**
# Summary
Adding mongoDB as destination for data integration pipelines

# Tests
Tested with postgreSQL source on MongoDB destination

Both FULL_TABLE and INCREMENTAL were tested, sync example:
Screenshot from 2023-07-25 16-10-18

MongoDB data example:
Screenshot from 2023-07-25 16-06-22

MongoDB document example:
Screenshot from 2023-07-25 16-24-06

Array example:
Screenshot from 2023-07-26 19-44-12

Screenshot from 2023-07-26 19-41-14

Object example:
Screenshot from 2023-07-26 19-44-42

Screenshot from 2023-07-26 19-41-27

PostgreSQL schema:
Screenshot from 2023-07-24 15-20-19

Screenshot from 2023-07-24 15-20-34

cc:
@wangxiaoyou1993

### CODE REVIEW

**Comment by wangxiaoyou1993** [comment]
Could you also test array type and object type?

File: mage_integrations/mage_integrations/destinations/mongodb/__init__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import argparse
+import sys
+
+from mage_integrations.destinations.base import Destination
+from mage_integrations.destinations.mongodb.target_mongodb.target import TargetMongoDb
+
+
+class Mongodb(Destination):
```
Comment (wangxiaoyou1993) [inline mage_integrations/mage_integrations/destinations/mongodb/__init__.py:8]: nit: Let's capitalize D, `MongoDb`

**Review by wangxiaoyou1993 — COMMENTED** [review]
Can you add README file?

**Comment by Luishfs** [comment]
> Could you also test array type and object type?

yes, will be adding it to the test table for future tests

**Review by wangxiaoyou1993 — APPROVED** [review]

**Pull request merged**